app/src/main/java/com/asfoundation/wallet/viewmodel/TransactionsViewModel.java

```diff
@@ -383,4 +409,16 @@ private void dismissNotification(CardNotification cardNotification) {
     disposables.add(transactionViewInteract.dismissNotification(cardNotification)
         .subscribe(this::fetchCardNotifications, this::onError));
   }
+
+  public void showSupportScreen() {
+    supportInteractor.displayChatScreen();
+  }
+
+  private void registerSupportUser(String walletAddress) {
+    supportInteractor.registerUser(walletAddress);
+  }
+
+  private void logoutSupportUser() {
+    supportInteractor.logoutUser();
+  }
```

Comment:
**Discussion**  This is just my opinion and I'm not sure if it's correct or not but do we really need this 3 methods?
Since we're doing only one operation on each couldn't we call it in the source directly?

Reply:
Agree, I think we should only use these when these actions require more than just the call to the interactor

Reply:
Agree, I think we should only use these when these actions require more than just the call to the interactor

Reply:
I prefer to have the abstraction of what is called or not on the view model to avoid changing the view implementation when the logic requires additional changes